### ISSUE TITLE
Can I keep my pdf file and the cards I captured from it in sync?

### ISSUE THREAD

**Issue by comienzo2093** [fonol/anki-search-inside-add-card#218]
First of all, I apologize for my lack of communication skills in English. I'm not sure what I'm trying to tell you will be conveyed exactly.

You are the best developer. I captured and moved the image from a pdf file, which was really hard work. However, using this add on made it really easy. You saved me a lot of time preparing for the exams, and you changed my life. Thank you very much.

I have one question. Can I keep my pdf file and the notes I captured from it in sync?

The pdf file I use is hundreds of pages. When some of them are modified, it is difficult to edit the cards created by capturing them one by one.

It would be nice if there was a feature that automatically change the cards when I changed the pdf.

The steps I'm talking about are:

1. edit pdf (with ios note app)

2. replaces sorce pdf

3. The added cards are automatically(Or use the'sync' button) updated (synchronized).

I ask you that this is a possible feature because it captures a specific part of the pdf. 
If not possible, can I link the added card and the corresponding page of the pdf with tags, etc.?

I would like to thank you again for your hard work, and I will start sponsoring patron regularly in the near future.

For me, English is not my first language, I am sorry that it is difficult to write my gratitude in writing.

Have a nice day.

**Comment by fonol** [comment]
Hi, 
> It would be nice if there was a feature that automatically change the cards when I changed the pdf.

Sorry, but that sounds like a gigantic task and far out of the scope of this add-on. It only links the cards you created to the page of the pdf where you were when you created them. The add-on in general is pretty dumb, it only knows you have a pdf file at a certain path on your computer, and that there are some cards you created on page x of that pdf file. It doesn't store anything about the content of the pdf file, so it wouldn't know when/what you changed in the file from outside.

> If not possible, can I link the added card and the corresponding page of the pdf with tags, etc.?

You mean like tagging a single link between a created note and a pdf page? I am curious, what would that be useful for, that you couldn't accomplish by tagging the notes you created on the page themselves?

**Comment by comienzo2093** [comment]
Hi,

Thank you for answer. 
When I thought about it again after posting, I realized that the feature improvement request I made here was unreasonable.

"It only links the cards you created to the page of the pdf where you were when you created them."

<<<<<<<<<<<<<<<<<<

oh.. I just found out that add on has this feature!

'If not possible, can I link the added card and the corresponding page of the pdf with tags, etc.?'

Therefore, this request is no longer valid. It was already implemented in a better way for this add-on.




It's a pity that the implementation of the features I'm talking about is impossible, but this addon is still attractive.

For example, I put hundreds of pages of pdfs on my ipad, edit them with a note app, capture the necessary content and add them to anki card.

And later, when I revised some of the pdf content, I reflected the modified content on the anki card.
This process was very cumbersome.

1. Move the pdf file back from ipad to pc, 
2. open it in pdf reader and find the modified page
3. capture it again and find right anki card
4. update it on that anki card.

This addon intuitively shows me which anki card I made on which pdf page. This makes the process simpler.
(Specifically, it shortens the time it takes to do steps 3 and 4.)

I'm sorry that I asked for an impossible feature because I didn't know much about programming. And I'll always be grateful for this add-on.

Thanks for the quick reply and have a nice day.
